src/kernel/chainparams.cpp

```diff
@@ -191,6 +191,12 @@ class CMainParams : public CChainParams {
                 .hash_serialized = AssumeutxoHash{uint256{"a2a5521b1b5ab65f67818e5e8eccabb7171a517f9e2382208f77687310768f96"}},
                 .m_chain_tx_count = 991032194,
                 .blockhash = consteval_ctor(uint256{"0000000000000000000320283a032748cef8227873ff4872689bf23f1cda83a5"}),
+            },
+            {
+                .height = 880'000,
```

Comment:
At the release we'll roughly be at 890'000, right? A bit tight, I guess, but no strong objections.

Reply:
We want this to be updated in master, so we have to pick something before the v29 branch-off.